tests/run-macros/i6253-c/quoted_2.scala

```diff
@@ -0,0 +1,10 @@
+import Macros._
+
+object Test {
+
+  def main(args: Array[String]): Unit = {
+    println(xyz"Hello World")
+    println(xyz"Hello ${"World"}")
```

Comment:
It's not obvious why both print `ERROR`, why do both matches fail?

Reply:
I will add a comment in the test. It is because the parameter was not marked as inline and we are matching against the binding.